setup.py

```diff
@@ -43,7 +43,7 @@
 api_requirements = [
     "Flask >=2.2.3, < 3.0",
     "Flask-Cors >=3.0.10, < 4.0",
-    "gunicorn >=20.1.0, < 21.0",
+    "gunicorn >=20.1.0, < 22.0",
```

Comment:
If the error was fixed in the latest major, I propose this:

```suggestion
    "gunicorn >=21.0, < 22.0",
```

Excepting `numpy`, it is preferable to avoid requirements spanning more than one major.

Reply:
Also, now that I think of it, without the contraint, if another library happens to require a lower version of gunicorn, the `pip` resolver can silently decide to use a version where the bug is not yet corrected.

Reply:
Thx, I made the change